.github/workflows/wheel_linux_x86_64_cuda.yml

```diff
@@ -22,6 +22,7 @@ concurrency:
 
 jobs:
   set_wheel_build_matrix:
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'ci:build_wheels') }}
```

Comment:
Wait, will this still run when merging to master if we don't have the label selected?

Reply:
AFAIK you won't be allowed to merge since there are required checks on some wheel workflows. So you'd need at least one commit to trigger a CI run after adding the label.

Reply:
But someone could remove the label after running the CIs. Right?

Reply:
We should make it independent of the label to run it when merging.

Reply:
> AFAIK you won't be allowed to merge since there are required checks on some wheel workflows. So you'd need at least one commit to trigger a CI run after adding the label.

@vincentmr let's double-check this before merge please.

Reply:
Yes, we can see it at [aeb77d5](https://github.com/PennyLaneAI/pennylane-lightning/pull/648/commits/aeb77d558ff16a5077532dd85a1d2065ebc1385c) for example.

Reply:
:partying_face: